src/tools/ceph_monstore_tool.cc

```diff
@@ -794,6 +871,8 @@ int main(int argc, char **argv) {
   g_ceph_context->_conf->apply_changes(NULL);
   g_conf = g_ceph_context->_conf;
 
+  if (check_lock(store_path))
+    return 1;
```

Comment:
we should return something of value to the user. maybe propagate the return from `check_lock()` as the program's return code?